src/pgroonga.c

```diff
@@ -5489,9 +5489,15 @@ PGrnSearchIsMatchInCondition(ScanKey key)
 {
 	return (key->sk_flags & SK_SEARCHARRAY) &&
 		   ((key->sk_strategy == PGrnMatchStrategyNumber) ||
-			(key->sk_strategy == PGrnMatchStrategyV2Number));
+			(key->sk_strategy == PGrnMatchStrategyV2Number) ||
+			(key->sk_strategy == PGrnLikeStrategyNumber) ||
+			(key->sk_strategy == PGrnILikeStrategyNumber));
 }
```

Comment:
Please don't reuse this function. This is for match condition (`&@  (...)`) not `LIKE`/`ILIKE`.

BTW, we should use `Any` not `In` for this function name... Because `COLUMN OPERATOR IN (...)` is invalid syntax. `COLUMN OPERATOR ANY (...)` is valid.

Reply:
I don't reuse this function. I created functions for `LIKE` and `ILIKE` at https://github.com/pgroonga/pgroonga/pull/917/commits/afe2f5d98d3138c3d50c7586e1dda558d1715be8.

Modify of this function name  will be fixed in a separate task.